packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -0,0 +1,77 @@
+const Logger = require('../logger/logger');
+
+const logger = new Logger({ prefix: 'wdio-terra-service' });
+
+/**
+* Convenience method for getting the Terra test viewports by name.
+*
+* @param {string|string[]} sizes - the list of Terra viewport names.
+* @return {object} - Terra viewport sizes.
+*/
+const getViewports = (...sizes) => {
+  const { terraViewports } = global.browser.config;
+  let viewportSizes = Object.keys(terraViewports);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(size => terraViewports[size]);
+};
+
+/**
+* Sets the viewport for the test run to the Terra test viewport size.
+*
+* @param {string} [formFactor=huge] - the Terra test viewport.
+*/
+const setViewport = (formFactor) => {
```

Comment:
If default is huge, should this be `(formFactor = 'huge')`? This would remove the if-else

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/5cf714558e6763787a83d19543932a2f57bc0c76